<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qunoitchi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widgetScan" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="cbPortList"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnConnect">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTextEdit" name="teRecvData">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>611</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>611</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="leReqData"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>tabWidget</zorder>
   <zorder>widgetScan</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Qunoitchi</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Serial</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionOption"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuPort"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Option</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
